frontend/src/components/Nav/Navigation.tsx

```diff
@@ -155,7 +155,7 @@ export class NavigationComponent extends React.Component<PropsType, NavigationSt
 
 const mapStateToProps = (state: KialiAppState) => ({
   navCollapsed: state.userSettings.interface.navCollapse,
-  jaegerUrl: state.jaegerState.info && state.jaegerState.info.url ? state.jaegerState.info.url : undefined
+  jaegerUrl: state.tracingState.info && state.tracingState.info.url ? state.tracingState.info.url : undefined
```

Comment:
If the state property is generic (tracingState), the variable should be renamed to tracingUrl